### DISCUSSION TITLE
¯\_(ツ)_/¯

### DISCUSSION THREAD

**Discussion by Boopath1** [projectdiscovery/nuclei#3896] [Q&A] [answered]
I don't know why the scanning is skipped, can you tell us why this feature is implemented? And is there any flag to bypass this feature? please let me know. I'm adding a screenshot for your perusal, please look at it.

![image](https://github.com/projectdiscovery/nuclei/assets/54469521/31892097-b8a0-4ad5-8d0c-b80f980c0b8c)

**Comment by ehsandeep** [comment, marked as answer]
@Boopath1 

```
   -nmhe, -no-mhe  disable skipping host from scan based on errors
```

This feature ensures skipping unresponsive hosts from scan, you can disable it using above option.
